site-manage/site-manage-tool/tool/src/webapp/vm/sitesetup/chef_site-modifyENW.vm

```diff
@@ -31,13 +31,16 @@
 	
 	## customize lti tools
 	#if (!$!ltiTools.isEmpty())
-		<p class="instruction">
-			$tlang.getString("ltiTools.custom.instruction")
-		</p>
 		#foreach($ltiTool in $ltiTools.entrySet())
 			#set($ltiToolId=$ltiTool.getKey())
 			#if (!$!oldLtiTools.containsKey($ltiToolId))
-				## show the input only if the tool is not enabled in the site yet
+				## show the input only if the tool is not enabled in the site yet and display the heading only once
+				#if (!$!instruction)
```

Comment:
I think the $! means no exception if instruction is null and the first ! is a logical not.